plugins/ipam/host-local/config.go

```diff
@@ -39,31 +39,29 @@ type IPAMArgs struct {
 }
 
 type Net struct {
-	Name string      `json:"name"`
-	IPAM *IPAMConfig `json:"ipam"`
+	Name  string      `json:"name"`
+	IPAM  *IPAMConfig `json:"ipam"`
+	IPAM6 *IPAMConfig `json:"ipam6"`
```

Comment:
Shouldn't at least one of these be omit-empty?  Actually I think both of them have to be omit-empty, but then the code needs to check that at least one of them is present and error out if not.  Otherwise adding ipam6 will require all network config to pass both ipam + ipam6, and nobody is passing ipam6 right now.